CONTRIBUTING.md

```diff
@@ -42,7 +42,7 @@ terraform {
 }
 
 provider "onepassword" {
-  url = "http://<1Password Connect API Hostname>"
+  service_account_token = "<1Password service account token>"
```

Comment:
`[question]`
While this promotes using the provider with a service account, we should mention the other possible ways of configuring the provider: either as minimized code examples or link to the relevant documentation.

The reason for this is because the contributors may not use service accounts in their flow, but rather one of the other configurations and it would help to share them here to have a smooth onboarding for any type of configuration.

Reply:
+1

Reply:
More details on how to setup the provider users can find in [Terraform Registry docs](https://github.com/1Password/terraform-provider-onepassword/pull/docs/index.md) and [examples](https://github.com/1Password/terraform-provider-onepassword/pull/examples/README.md)

The docs above are the source of true and entry point. The other examples show one of the possible ways of setting it up.
The benefit of that would be to reduce the amount of docs we'll need to support which makes it clean. Also users have to read less 😄

Let me know if that makes sense.

Reply:
I'm not suggesting to copy-paste the documentation for the other configuration options. Adding a link to them like this should suffice:

> Check [this documentation](inset-link-here) for more details about provider configuration.


This doesn't increase a lot the size of the documentation, while also ensuring that the user knows where to reach other documentation (rather than us expecting them to look for the place where it is e.g.  [Terraform Registry docs](https://github.com/1Password/terraform-provider-onepassword/pull/docs/index.md)). This improves the user journey in terms of learning how to use the tool to be a great contributor. We shouldn't assume that they know where to find the info. The last advantage of this (which can be nitpicky) is that this results of the user making one click to reach to the right documentation rather than....at least 3-5 with the other approach (i.e. getting to the docs on their own).

Anyway, we can always iterate on this, so I'm fine with it as it is now. 😄